airflow/www/static/js/tree/InstanceTooltip.jsx

```diff
@@ -24,30 +24,33 @@ import { Box, Text } from '@chakra-ui/react';
 
 import { formatDateTime, getDuration, formatDuration } from '../datetime_utils';
 
+const STATES = [
+  ['success', 0],
+  ['failed', 0],
+  ['upstream_failed', 0],
+  ['up_for_retry', 0],
+  ['up_for_reschedule', 0],
+  ['running', 0],
+  ['deferred', 0],
+  ['sensing', 0],
+  ['queued', 0],
+  ['scheduled', 0],
+  ['skipped', 0],
+  ['no_status', 0],
```

Comment:
Not thrilled about having this list in multiple JS files. Is there any sensible way to define it once and reuse it?

Reply:
Made a `utils.js` file for this and other similar uses.